eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/shared/popUp.js

```diff
@@ -0,0 +1,25 @@
+﻿(function (angular) {
+    angular.module('equizModule')
```

Comment:
Wrong syntax of a directive declaration.
Use quizReview-data.service.js as an example